projects/app1/src/app/app.component.css

```diff
@@ -0,0 +1,9 @@
+* {
+  background-color: red;
+    }
+
+div {
+  background: green;
+    }
+
+.test {}

```

Comment:
**Suggestion:** Empty CSS rulesets should be avoided. If the '.test' class is not being used, consider removing it.
```suggestion

```